src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.c

```diff
@@ -618,14 +597,12 @@ void func_80896D78(BgJyaCobra* this, GlobalContext* globalCtx) {
 
 void BgJyaCobra_DrawShadow(BgJyaCobra* this, GlobalContext* globalCtx) {
     u32 pad;
```

Comment:
```suggestion
    s32 pad;
```
Unless it needs `u32`